lua/entities/gmod_wire_expression2/core/custom/cfc_trigonometry_geometry.lua

```diff
@@ -0,0 +1,38 @@
+E2Lib.RegisterExtension( "cfc_e2_lib", true )
```

Comment:
This file needs to be its own PR. I won't review it in here.

Reply:
How do i split the pr up?

Reply:
create a new branch and pr with this file and remove it from this branch

Reply:
k thanks, i will do it when i have the time